sql_generators/stable_views/__init__.py

```diff
@@ -281,6 +301,23 @@ def write_view_if_not_exists(target_project: str, sql_dir: Path, id_token=None,
             ),
             trailing_newline=True,
         )
+    #For views without app_display_version, use the template without app version parsing
```

Comment:
`client_info` is a glean field so this would break all other views.  You can make use of `schema.schema_id == "moz://mozilla.org/schemas/glean/ping/1"` like on line 186 above so that `client_info.app_display_version` is only used on those tables.

For non-glean, `application.display_version` is used in some places but not everywhere so you would need to have a lot of special cases.  I would consider just scoping this change to glean views

Reply:
Thanks @BenWu - will do!! :)